tests/integration/test_repeat.py

```diff
@@ -23,42 +22,183 @@
 np.random.seed(12345)
 
 
-def test_basic():
-    assert np.array_equal(num.repeat(3, 4), np.repeat(3, 4))
-    assert np.array_equal(num.repeat([3, 1], 4), np.repeat([3, 1], 4))
+@pytest.mark.parametrize(
+    "array", (None, [], 4, [2, 3], mk_seq_array(num, (3, 4, 2)))
+)
+def test_repeats_none(array):
+    with pytest.raises(TypeError):
+        num.repeat(array, None)
```

Comment:
@manopapad what do you think about including a numpy branch to these exception tests:
```suggestion
def test_repeats_none(array):
    with pytest.raises(TypeError):
        np.repeat(array, None)
    with pytest.raises(TypeError):
        num.repeat(array, None)
```
My reasoning is that what we ultimately want from the tests is an assertion that behavior matches numpy, and without a numpy branch, all we achieve is maintaining the cunumeric behaviour, without maintaining it matches numpy.

Reply:
That makes sense

Reply:
:+1: @xialu00 don't worry about changing anything in this PR, there are lots of these all over, I will go through separately and see if there are any helper functions or fixtures that can be added to perform these checks in a consistent way.